code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -908,4 +929,18 @@
 	S["phrases"] 			<< speech_buffer
 	S["roundssurvived"]		<< rounds_survived
 	S["longestsurvival"]	<< longest_survival
+	S["longestdeathstreak"] << longest_deathstreak
 	memory_saved = 1
+
+/mob/living/simple_animal/parrot/Poly/ghost
+	name = "The Ghost of Poly"
+	desc = "Doomed to squawk the earth."
+	color = "#FFFFFF77"
+	speak_chance = 20
+	status_flags = GODMODE
+	incorporeal_move = 1

```

Comment:
A* doesn't account for incorpreal move but poly can still get in situations where he could use it because he looks for paths with A*, but then moves straight to whatever he's targeting.